src/storage/mod.rs

```diff
@@ -2713,6 +2723,53 @@ mod tests {
                 .async_get(Context::default(), Key::from_raw(b"x"), 105)
                 .wait(),
         );
+
+        let mut options = Options::default();
```

Comment:
I think the test below could be placed in a separate function. Not only because it is just a test case independent of the previous one, but also to avoid confusing people by mixing _version_ and _physical time_.